build_system/src/build.rs

```diff
@@ -184,8 +184,9 @@ pub fn build_sysroot(env: &HashMap<String, String>, config: &ConfigInfo) -> Resu
 fn build_codegen(args: &mut BuildArg) -> Result<(), String> {
     let mut env = HashMap::new();
 
-    env.insert("LD_LIBRARY_PATH".to_string(), args.config_info.gcc_path.clone());
-    env.insert("LIBRARY_PATH".to_string(), args.config_info.gcc_path.clone());
+    let gcc_path = args.config_info.gcc_path.clone().unwrap();
```

Comment:
Please use `expect()` here and in the other places or add a comment to explain that this cannot happen since an error would be emitted in `config.rs` if the path is not specified somehow.

Reply:
I forgot to save the file... XD